datalad/metadata/metadata.py

```diff
@@ -357,7 +444,7 @@ def __call__(
                     db[k] = v
                 for k, v in add.items():
                     db[k] = sorted(unique(
-                        db.get(k, []) + v))
+                        db.get(k, []) + list(v)))
```

Comment:
indeed, assure_list would be safer. Not sure about the "coffee"  -- I'll stick with beer.